packages/sockjs-shim/server.js

```diff
@@ -0,0 +1,29 @@
+import { onPageLoad } from "meteor/server-render";
+
+const sockjsVersion = "0.3.4";
+const scriptPath =
+  "/packages/sockjs-shim/sockjs-" +
+  sockjsVersion +
+  (Meteor.isProduction ? ".min.js" : ".js");
+
+const hasOwn = Object.prototype.hasOwnProperty;
+const minimumMajorVersions = {
+  chrome: 16,
+  firefox: 11,
+  ie: 10,
+  safari: 7,
```

Comment:
This list is probably not complete, and I'm open to adjusting these versions. I took this information from [CanIUse](https://caniuse.com/#feat=websockets).

Reply:
One significant missing browser: `phantomjs: 2`. This means `meteor test-packages` is still running with SockJS polyfilled, not using native WebSockets. For the foreseeable future, I think we need to run these tests with and without SockJS, to ensure both approaches work.